### PR TITLE
LTP: created and updated environments

### PR DESCRIPTION
**Pull request by nareshkamboju**
For easy readability new environments have been created and updated.
Following environments have been created
 - environments_qemu_all
 - environments_qemu_arm_all
 - environments_intel_all
 - environments_32bit_all
 - environments_arm64_arm32

Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>

### CODE REVIEW

**Comment by danrue** [comment]
So we have the following environment lists:
- environments_all
- environments_arm64
- environments_arm32
- environments_x86_64
- environments_i386
- environments_qemu_all
- environments_32bit_all
- environments_qemu_arm_all
- environments_intel_all
- environments_arm64_arm32

I think it'd be a bit more consistent if you dropped the _all from environments_qemu_all, environments_32bit_all, environments_qemu_arm_all, and environments_intel_all.

File: ltp-production.yaml [hunk 1]
```diff
@@ -236,14 +223,7 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=3318
     active: true
     intermittent: false
-  - environments:
-    - hi6220-hikey
-    - juno-r2
-    - qemu_x86_64
-    - qemu_arm64
-    - dragonboard-410c
-    - qemu_i386
```
Comment (danrue) [inline ltp-production.yaml]: Was it intentional to drop i386 from this known issue? I noticed when reading the build output @ https://travis-ci.com/Linaro/qa-reports-known-issues/builds/84374661

Comment (nareshkamboju) [reply ltp-production.yaml]: Good catch !
I have intentionally dropped i386 because it is getting pass on i386 confirmed from staging results.

**Comment by danrue** [comment]
Merged. Thanks Naresh!

**Pull request merged**